integration/meta-test.ts

```diff
@@ -419,7 +419,6 @@ test.describe("v2_meta", () => {
             { name: "description", content: data.description },
             { property: "og:image", content: "https://picsum.photos/200/200" },
             { property: "og:type", content: data.contentType }, // undefined
-            { httpEquiv: "refresh", content: "3;url=https://www.mozilla.org" },
```

Comment:
why did this one get removed? 😬

Reply:
Because it was super annoying to debug with `app.poke` because the browser kept redirecting me to mozilla.org 😂 

I don't mind adding it back, just pulled it out while working through some issues. That said, I think it's a little less necessary to test various individual properties with the new approach since we're no longer doing anything special for `"refresh"`.

Reply:
Oh, I'm fine with removing a test because it works too well 😅